modules/earthengine-layers/src/earth-engine-mesh-layer.js

```diff
@@ -1,7 +1,6 @@
 /* global createImageBitmap */
 import {CompositeLayer} from '@deck.gl/core';
-import {TerrainLayer} from '@deck.gl/geo-layers';
-import TileLayer from './tile-layer/tile-layer';
```

Comment:
Here I removed the usage of the local `TileLayer`, but forgot to delete the copy.